packages/core/src/common/commands-common.ts

```diff
@@ -9,6 +9,16 @@ import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
 
+const enjson = require('./i18n/en.json');
+const esjson = require('./i18n/es.json');
+const frjson = require('./i18n/fr.json');
+const Globalize = require("globalize");
+
+// Load potential languages
+Globalize.loadMessages(enjson);
+Globalize.loadMessages(esjson);
+Globalize.loadMessages(frjson);
```

Comment:
No, I do not. We could maybe generate a library function together with the json files that we generate from translating transifex files (or something similar)?